src/Core/Import/Exception/NotSupportedImportEntityException.php

```diff
@@ -24,12 +24,13 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
-header('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
+namespace PrestaShop\PrestaShop\Core\Import\Exception;
 
-header('Cache-Control: no-store, no-cache, must-revalidate');
-header('Cache-Control: post-check=0, pre-check=0', false);
-header('Pragma: no-cache');
+use Exception;
 
-header('Location: ../../../../../../../');
-exit;
+/**
+ * Class NotSupportedImportEntityException is thrown when not supported import entity is provided.
+ */
+class NotSupportedImportEntityException extends Exception implements ExceptionInterface
```

Comment:
You can have a look at [Categories PR](https://github.com/PrestaShop/PrestaShop/pull/10408/files#diff-f5fcc1d22a202d955506ff8c005d86f9R35) to see our Exception hierarchy